### ISSUE TITLE
De-Fusion: Tonesque

### ISSUE THREAD

**Issue by sdixon194** [Automattic/jetpack#25239]
### Impacted plugin

Jetpack

### What

Tonesque

### How

```
"$plugin_root/_inc/lib/class.color.php" => "$wpcom_root/wp-content/lib/class.color.php",
"$plugin_root/_inc/lib/tonesque.php"    => "$wpcom_root/wp-content/lib/tonesque.php",
```

**Comment by anomiex** [comment]
Only one wpcom diff needed: D87739-code

Then the Jetpack cleanup: #26221

**Comment by sdixon194** [comment]
Themes that support colors still work for me 👍

**Comment by anomiex** [comment]
Phab diff merged, just waiting on review of #26221 now.
